common/lib/xmodule/xmodule/contentstore/content.py

```diff
@@ -38,11 +38,18 @@ def __init__(self, loc, name, content_type, data, last_modified_at=None, thumbna
         # cycles
         self.import_path = import_path
         self.locked = locked
+        self.content_digest = content_digest
 
     @property
     def is_thumbnail(self):
         return self.location.category == 'thumbnail'
 
+    def __str__(self):
+        return 'StaticContent(loc={}, name={}, type={}, length={}, locked={}, digest={})'.format(
+            self.location, self.name, self.content_type, self.length,
+            self.locked, self.content_digest
+        )
```

Comment:
1. When does this get called?
2. Please test that this doesn't explode if assets have non-ASCII characters (you can upload files with non-Latin file names and those names will be preserved).